common-rinkeby.env

```diff
@@ -0,0 +1,3 @@
+STABLEX_CONTRACT_ADDRESS=0x9046451F7cF124c1d7d1832F76F5e98a33D1610E
```

Comment:
#271 added a few more variables that are needed for rinkeby. We probably don't want to expose the PK but at least the NETWORK_ID